src/pyhf_benchmark/run.py

```diff
@@ -1,114 +1,15 @@
-import requests
-import tarfile
-import os
-import json
 import click
-import shutil
 import pyhf
 import warnings
 from datetime import datetime
-from pathlib import Path
+import pyhf_benchmark.load as load
+import pyhf_benchmark.mle as mle
```

Comment:
Same thing as above. Importing packages from inside themselves is a problem. Just use relative imports. (Search on this if you're not familiar)